src/components/Charts/ChartModule.style.js

```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+
+import { COLORS } from 'enums';
+import { colors } from '@material-ui/core';
+
+export const ChartContainer = styled.section`
+  width: 100%;
+`;
+
+export const ChartHeader = styled.div`
+  max-width: 900px;
+  padding: 1.5rem 0;
+  text-align: left;
+
+  span {
+    color: rgba(0, 0, 0, 0.7);
+  }
+
+  h2 {
+    font-weight: 700;
+  }
+
+  @media (max-width: 600px) {
+    padding: 1.5rem 1rem;
+  }
+`;
+
+export const Wrapper = styled.div`
+  max-width: 900px;
+  margin: auto;
+
+  .highcharts-axis-labels {
```

Comment:
We will probably move the chart styles to its own `style` file, but we can leave it for now.

Reply:
cook gonna ack this for right now because i moved everything everywhere but yes agree and will do in an upcoming PR